inc/qrcode.class.php

```diff
@@ -233,6 +245,11 @@ static function processMassiveActionsForOneItemtype(MassiveAction $ma, CommonDBT
                }
             }
             if ($ma->POST['type'] == 'QRcode') {
+               if($item->isField('inventotynumberURL')) {
+                  $URLtype = 'inventoryURL';
+               } else {
+                  $URLtype = 'idURL';
+               }
```

Comment:
This code looks useless. What was it supposed to do ?

1. `inventotynumberURL` has a typo.
2. `$URLtype` is not used.